frost-core/src/frost/keys.rs

```diff
@@ -233,13 +233,17 @@ pub struct VerifiableSecretSharingCommitment<C: Ciphersuite>(
     pub(super) Vec<CoefficientCommitment<C>>,
 );
 
-/// A secret share generated by performing a (t-out-of-n) secret sharing scheme.
+/// A secret share generated by performing a (t-out-of-n) secret sharing scheme,
+/// generated by a dealer performing [`keygen_with_dealer`].
```

Comment:
We'll probably update this when the DKG is merged?

Reply:
DKG can't use the same struct, because it requires more data to verify the share :cry: (at least I couldn't figure it out)

Reply:
Gotcha